pipelines/build-test-tool-template.yaml

```diff
@@ -17,7 +17,23 @@ steps:
     inputs:
       arguments: '-c $(BuildConfiguration)'
 
-  - task: DotNetCoreCLI@2
-    displayName: Run tests
+  - pwsh: |
+      dotnet test -c $(BuildConfiguration) --no-build -- --report-trx --results-directory $(Build.ArtifactStagingDirectory)/artifacts/TestResults --coverage --coverage-output $(Build.ArtifactStagingDirectory)/artifacts/Coverage/coverage.cobertura.xml --coverage-output-format cobertura --hangdump --hangdump-filename hangdump.dmp
```

Comment:
Why have you moved out of `DotNetCoreCLI`?

Reply:
Any advantage to use `DotNetCoreCLI`? I think both can work

Reply:
Sure both work. I usually go with using the task when it's available compared to manual "code".